DATA605/project_description.py

```diff
@@ -4,44 +4,117 @@
+import pandas as pd
+
+_LOG = logging.getLogger(__name__)
 
-# Constants
+# Constants.
 GOOGLE_SHEET_URL = "https://docs.google.com/spreadsheets/d/1Ez5uRvOgvDMkFc9c6mI21kscTKnpiCSh4UkUh_ifLIw/edit?gid=0#gid=0"
-PROMPT_DOC_URL = "https://docs.google.com/document/d/1c4oyCmvb-r5FHHL0enRQEEfsQ-shR3F7uvFVXMdeJSg/edit?tab=t.0#heading=h.a4upg2qbwqkb"
-MARKDOWN_FILE_PATH = "DATA605_Projects.md"
+PROMPT_DOC_URL = """
+You are a college professor of Data science. In the next prompt I will give you a topic XYZ for a class project and you will write a description using bullet points for a college class project about implementing an example big data system in Python.
+
+The project should be related to ingesting and processing real-time data about bitcoin. The focus should be on the technology XYZ, using basic Python packages for anything else.
+
+The assignment requires to describe the basic functionalities of the package using examples and then a concrete project related to implementing something related to time series analysis.
+The complexity of the project is 1, where 1 is easy (it should take around 7 days) to develop, 2 is medium difficulty (it should take around 10 days to complete), 3 is hard (it should take 14 days to complete).
+
+The output should follow the template below
+Title:
+Difficulty: (1=easy, 3=difficult)
+Description
+Describe technology
+Describe the project
+Useful resources
+Is it free?
+Python libraries / bindings
+
+An example of a project where XYZ is “AWS Glue” is the following
+
+Title: Ingest bitcoin prices using AWS Glue
+Description
+AWS Glue is a fully managed extract, transform, and load (ETL) service provided by Amazon Web Services (AWS). It helps you prepare and transform data for analytics, machine learning, and other data processing tasks. Understand AWS Glue's core concepts, including data catalogs, crawlers, and jobs
+An easy project that can be completed in one week involves using AWS Glue to ingest Bitcoin price data from a public API, such as CoinGecko, and store it in an S3 bucket for further analysis. The first step of the project is to set up an AWS Glue crawler to automatically discover and catalog the incoming JSON data. Students will configure the crawler to run on a schedule to continuously fetch new Bitcoin price data from the API every few minutes. They will store the data in an S3 bucket and create a Glue Data Catalog to manage the dataset’s schema. Then, a simple AWS Glue job can be written using Python (with PySpark) to process and transform the raw data, such as filtering the data for specific time intervals or adding new fields, like the price change over time. Finally, the processed data can be stored back in S3 in a more structured format (e.g., Parquet) for easy querying. This project gives students hands-on experience with AWS Glue and basic ETL processes while learning how to handle real-time data ingestion.
+Difficulty: easy (it should take around one week)
+Useful links
```

Comment:
This is part is missing. We can even remove it IMO here and above (which it's called useful resources)